ci/ci/github.py

```diff
@@ -566,9 +572,14 @@ def is_up_to_date(self):
         return self.batch is not None and self.target_branch.sha == self.batch.attributes['target_sha']
 
     def is_mergeable(self):
+        if self.last_known_github_status[GITHUB_STATUS_CONTEXT] == 'success':
```

Comment:
heh, mypy isn't checking this method because of the lack of any type signature (try adding `-> bool`). This needs to be `== GithubStatus.SUCCESS`. Same below in the `all(..)` expression.